jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketSession.java

```diff
@@ -189,11 +189,18 @@ public void close() throws IOException
      * @since JSR356 v1.0
      */
     @Override
-    public void close(CloseReason closeReason) throws IOException
+    public void close(CloseReason closeReason)
     {
-        FutureCallback b = new FutureCallback();
-        coreSession.close(closeReason.getCloseCode().getCode(), closeReason.getReasonPhrase(), b);
-        b.block(getBlockingTimeout(), TimeUnit.MILLISECONDS);
+        try
+        {
+            FutureCallback b = new FutureCallback();
+            coreSession.close(closeReason.getCloseCode().getCode(), closeReason.getReasonPhrase(), b);
+            b.block(getBlockingTimeout(), TimeUnit.MILLISECONDS);
```

Comment:
`getBlockingTimeout()` is only used from `close()` so perhaps should be renamed to `getCloseTimeout()`?